docs/01_setup.md

```diff
@@ -119,19 +130,19 @@ See [MACI docs](https://maci.pse.dev/docs/quick-start/deployment#deployment-usin
 
 The `.env.example` file in the `packages/interface/` folder contains instructions for most of these steps.
 
-At the very minimum you need to configure a **subgraph**, **admin address**, **MACI address**, the **EAS Schema** and the **application registration periods** under App Configuration.
+At the very minimum you need to configure a **subgraph**, **admin address**, **MACI address**, the **EAS Schema** and the **application registration periods** under App Configuration. You can do this following these next steps.
 
 ### Subgraph
 
-In the **MACI** repo and head to the subgraph folder.
+Got to the **MACI** repo and head to the subgraph folder.
```

Comment:
Typo here

Reply:
actually there's subgraph folder in the `maci-platform` repo, no need to go to the **maci** repo